pkg/backends/locust/locust.go

```diff
@@ -0,0 +1,211 @@
+package locust
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+
+	coreV1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+
+	"github.com/hellofresh/kangal/pkg/core/helper"
+	loadTestV1 "github.com/hellofresh/kangal/pkg/kubernetes/apis/loadtest/v1"
+	clientSetV "github.com/hellofresh/kangal/pkg/kubernetes/generated/clientset/versioned"
+	"go.uber.org/zap"
+)
+
+var (
+	defaultImage    = "locustio/locust"
+	defaultImageTag = "latest"
```

Comment:
Should we pin the version here?

Reply:
I don't think we should PIN here, but in the helm chart definitely, I will change that

Reply:
Done!